spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
It cannot replace `SPC p p` for scope reason and for the less confortable UX to switch project which is obvious. `p` in spacemacs is for project scope and I won't mess with it.

In fact I don't want CtrlP in spacemacs, I want smartly put together sources with the following rules:
- the top source must be the one that the user wants to look for
- the other sources are fallback sources that make sense with the top source. 

If spacemacs was a Vim configuration and CtrlP the Helm of this configuration then it would have been the same thing: I would have used the more focused CtrlP commands on different bindings, not the mixed one. Or make smart compromises by choosing carefully how sources are associated.

Anyway I think my point is clear. I don't care about the number of key bindings, the important thing is documentation and discoverability. Respecting the scopes are hugely important.